src/I18n/DateFormatTrait.php

```diff
@@ -157,9 +201,22 @@ public function i18nFormat($format = null, $timezone = null, $locale = null)
     {
         $time = $this;
 
+        if ($time instanceof Time || $time instanceof FrozenTime) {
+            // Remove before merge:
+            // if ((is_string($time->timezone) && $time->timezone === 'UTC')
+            //     || ($time->timezone instanceof \DateTimeZone && $time->timezone->getName() === '+0:00')
+            // ) {
+            // }
+            // if ($time->getOffset() !== 0) {
+            //     $time = clone $time;
+            //     $time->add(new \DateInterval('PT' . $time->getOffset() / 60 . 'M'));
+            // }
```

Comment:
This is a dev note that's two different conditions I have been trying to get offsets + output time zones to work. If you take a look at the failing tests you will see the reason why. However I could not figure this last issue out.